### PR TITLE
Update Plugin Docs

### PR DESCRIPTION
**Pull request by napcae**
according to #4126

### CODE REVIEW

**Comment by envygeeks** [comment]
As I said in the other one, `jekyll-assets` stays.

File: site/_docs/plugins.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ Jekyll generates your site.
 values of the gem names of the plugins you'd like to use. An example:
 
 
-        gems: [jekyll-test-plugin, jekyll-jsonify, jekyll-assets]
+        gems: [jekyll-sass-converter, jekyll-coffeescript, jekyll-watch, jekyll-assets]
```
Comment (parkr) [inline site/_docs/plugins.md]: `jekyll-sass-converter` comes pre-packaged with `jekyll` so let's remove that one.

**Comment by parkr** [comment]
Thanks!

**Comment by parkr** [comment]
@napcae Just went to go merge this but there's a weird `t` file? After than, :shipit:

**Comment by napcae** [comment]
Note to myself: Don't drink and git. My bad, sorry :)

**Comment by parkr** [comment]
Thanks! One last request: will you please squash these into one commit?

**Comment by napcae** [comment]
Done then :+1:

**Pull request merged**